packages/analytics/analytics-metric-provider/src/components/MetricsConsumer.vue

```diff
@@ -39,13 +39,16 @@ const { traffic, latency } = providerData.data
 const { i18n } = composables.useI18n()
 
 const trafficCard = composables.useMetricCardBuilder({
+  cardType: MetricCardType.TRAFFIC,
   title: computed(() => providerData.longCardTitles
     ? i18n.t('metricCard.long.traffic')
     : i18n.t('metricCard.short.traffic')),
+  description: providerData.description,
   record: traffic.mapped,
   hasError: traffic.hasError,
   lookupKey: props.lookupKey,
   sumGroupedValues: ALL_STATUS_CODE_GROUPS,
+  trendRange: computed(() => providerData.trendRange.value),
```

Comment:
This `computed` isn't doing anything -- if you'd like a ref, you should just be able to pass `providerData.trendRange`.

Reply:
Well, i'll be damned... Thanks for the heads up.